.circleci/config.yml

```diff
@@ -0,0 +1,261 @@
+##################################################################################
```

Comment:
Moet die hele config daar in zitten? Is een test en lint niet voldoende in principe hier?

Reply:
Op zich maakt het niet zo heel erg veel uit ze trouwens. Misschien beter zo want anders gaan ze de config uit Slite overnemen en is uw yarn stuff kapot.

Reply:
Nah, you're right. Er staat heel veel in die hier niet moet staan (verwijzingen naar onze Docker Hub en Bitbucket bijvoorbeeld).